support-frontend/assets/components/subscriptionCheckouts/address/__tests__/addressFieldStoreTests.js

```diff
@@ -0,0 +1,44 @@
+import { isHomeDeliveryInM25 } from 'components/subscriptionCheckouts/address/addressFieldsStore';
+
+jest.mock('ophan', () => {});
+
+describe('addressFieldStore', () => {
+
+  describe('isHomeDeliveryInM25 - this is a form error check so when this function returns "false" when there is an error', () => {
+
+    it('should return true when the order is a home delivery and the postcode in the M25', () => {
```

Comment:
Big thumbs-up for the tests, although I think we are essentially testing `postcodeIsWithinDeliveryArea` in two test suites now.  Not a big deal, but if you wanted to isolate the unique functionality of `isHomeDeliveryInM25` you could just check whether `postcodeIsWithinDeliveryArea` is called or true is returned. In fact, it might be easier to test if the predicate for checking the postcode were passed in, rather than internal to `isHomeDeliveryInM25`.  I wouldn't go back and change this for this PR, just a thought for next time.